minecraft_servers/svanemc/manifest.json

```diff
@@ -0,0 +1,32 @@
+
+{
+  "server_name": "svanemc",
+  "nice_name": "SvaneMC.net",
+  "direct_ip": "svanemc.net",
+  "supported_languages": [
+    "dk",
+    "en"
+  ],
+  "social": {
+    "web": "https://svanemc.net",
+    "web_shop": "https://buycraft.svanemc.net",
+    "web_support": "https://gommehdnet.zendesk.com",
+    "discord": "https://discord.gg/svanemc"
+  },
+  "gamemodes": {
+    "Skyblock": {
+      "name": "Skyblock",
+      "command": "/server skyblock",
+      "versions": "1.21<*"
+    }
+  },
+  "discord": {
+    "server_id": 1278101104059613184,
+    "rename_to_minecraft_name": true
```

Comment:
`true` is reserved for LabyMod Partners.
Please change to `false`

Reply:
Hey Dojap,
Thank you for your feedback.

I've made the changes, but we are actually Labymod partners, - That's why it's still true.
Thanks again. :)

Reply:
Sorry, my bad!